zubhub_frontend/zubhub/src/assets/js/styles/components/button/buttonStyles.js

```diff
@@ -1,7 +1,7 @@
 const styles = theme => ({
   primaryButtonStyle: {
     backgroundColor: '#00B8C4',
-    borderRadius: 30,
+    borderRadius: '30px !important',
```

Comment:
you shouldn't be messing with the primaryButtonStyle or anything inside the buttonStyles.js file directly. This is probably better fixed by removing the button group entirely (+ or - some additional changes), not tweaking this file.

Reply:
sure, @NdibeRaymond point noted and i will make some changes in a way and make a new PR, thanks you so much for your instructions and review.